packages/promotion/src/scripts/migration-down.ts

```diff
@@ -0,0 +1,42 @@
+import { LoaderOptions, Logger, ModulesSdkTypes } from "@medusajs/types"
+import { DALUtils, ModulesSdkUtils } from "@medusajs/utils"
+import { EntitySchema } from "@mikro-orm/core"
+import * as PromotionModels from "@models"
+
+/**
+ * This script is only valid for mikro orm managers. If a user provide a custom manager
+ * he is in charge of reverting the migrations.
+ * @param options
+ * @param logger
+ * @param moduleDeclaration
+ */
+export async function revertMigration({
+  options,
+  logger,
+}: Pick<
+  LoaderOptions<ModulesSdkTypes.ModuleServiceInitializeOptions>,
+  "options" | "logger"
+> = {}) {
+  logger ??= console as unknown as Logger
+
+  const dbData = ModulesSdkUtils.loadDatabaseConfig("promotion", options)!
```

Comment:
todo: `Modules.PROMOTION` The idea is to reuse as much as possible the enums and const we are creating and prevent usage of plane string which is prone to error and make a simple change difficult. It is not because it was done like this elsewhere that it was right and we must do some cleanup (part of the good camper process)